deploy/helm/sumologic/templates/logs/collector/otelcol/serviceaccount.yaml

```diff
@@ -7,10 +7,6 @@ metadata:
   labels:
     app: {{ template "sumologic.labels.app.logs.collector.serviceaccount" . }}
     {{- include "sumologic.labels.common" . | nindent 4 }}
-{{- if and .Values.sumologic.logs.collector.otelcloudwatch.enabled .Values.sumologic.logs.collector.otelcloudwatch.roleArn}}
-  annotations:
-    eks.amazonaws.com/role-arn: {{ .Values.sumologic.logs.collector.otelcloudwatch.roleArn }}
-{{- end }}
```

Comment:
Why are we removing this?

Reply:
`cloudwatch` has it own serviceaccount: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/drosiek-windows-logs-collector/deploy/helm/sumologic/templates/logs/collector/otelcol-cloudwatch/serviceaccount.yaml